client/src/main/java/io/pravega/client/segment/impl/AsyncSegmentInputStreamImpl.java

```diff
@@ -220,7 +220,7 @@ private void closeConnection(Exception exceptionToInflightRequests) {
         if (closed.get()) {
             log.info("Closing connection to segment: {}", segmentId);
         } else {            
-            log.info("Closing connection to segment {} with exception: {}", segmentId, exceptionToInflightRequests);
+            log.info("Closing connection to segment {} with exception: {}", segmentId, exceptionToInflightRequests.toString());
```

Comment:
I'm fine with the summarized version of the exception, but since we are dealing with an exception, shouldn't we make this a `warn` rather than info?

Reply:
Yes, it should be `warn` infact the 0.5 code has the following code snippet.

```
  private void closeConnection(Throwable exceptionToInflightRequests) {
        if (closed.get() || exceptionToInflightRequests instanceof ConnectionClosedException) {
            log.debug("Closing connection to segment {} with exception {}", this.segmentId, exceptionToInflightRequests);
        } else {
            log.warn("Closing connection to segment {} with exception: {}", this.segmentId, exceptionToInflightRequests);
        }
...
```